Dalamud/Game/Gui/ChatGui.cs

```diff
@@ -246,30 +288,32 @@ private void PrintTagged(string message, XivChatType channel, string? tag, ushor
 
         this.Print(new XivChatEntry
         {
-            Message = builder.AddText(message).Build(),
+            Message = builder.Build().Append(message),
```

Comment:
If this is just a `SeString`, why bother invoking the builder?

Reply:
I haven't touched this method, the git diff is just confusing it with the removed `string` one